cli/azd/pkg/convert/util.go

```diff
@@ -54,17 +56,51 @@ func ToMap(value any) (map[string]any, error) {
 		return nil, fmt.Errorf("failed to convert value to json: %w", err)
 	}
 
-	mapValue := map[string]any{}
+	var mapValue map[string]any
 	if err := json.Unmarshal(jsonValue, &mapValue); err != nil {
 		return nil, fmt.Errorf("failed to convert value to map: %w", err)
 	}
 
 	return mapValue, nil
 }
 
+// ToJsonArray converts the specified value (slice) to a json array
+func ToJsonArray(value any) ([]any, error) {
```

Comment:
Just wondering - do we have to do a round trip via JSON here?  Couldn't we just do a cast to `[]any`?

Reply:
We also want to make sure proper JSON marshalling happens to go from `PascalCase` to `camelCase` expected by the service.